### PR TITLE
CIV-11256 claimant dashboard status

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-11256

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimantClaimMatcher.java [hunk 1]
```diff
@@ -124,8 +129,14 @@ public boolean isPaperResponse() {
             return false;
         }
 
+        //return Objects.nonNull(caseData.getTakenOfflineDate()) && Objects.nonNull(caseData.getCcdState())
```
Comment (jarekPierchala) [inline src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimantClaimMatcher.java:132]: do we need to have it here if its been commented out?

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardDefendantClaimMatcher.java [hunk 2]
```diff
@@ -294,4 +294,14 @@ public boolean isClaimantDefaultJudgement() {
             && caseData.getPaymentTypeSelection() != null;
     }
 
+    @Override
+    public boolean isPartialAdmissionAccepted() {
+        return false;
```
Comment (jarekPierchala) [inline src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardDefendantClaimMatcher.java:299]: does it suppose to be hardcoded?

File: src/main/java/uk/gov/hmcts/reform/cmc/model/CmcClaim.java [hunk 3]
```diff
@@ -399,4 +399,23 @@ public boolean isPartialAdmissionRejected() {
...
+    public boolean isPartialAdmissionAccepted() {
+
+        return hasResponse() && response.isPartAdmitPayImmediately()
+            && !response.getPaymentIntention().hasAlreadyPaid()
+            && claimantAcceptedDefendantResponse();
+    }
+
+    @Override
+    public boolean isPaymentPlanAccepted() {
+        //return false;
```
Comment (jarekPierchala) [inline src/main/java/uk/gov/hmcts/reform/cmc/model/CmcClaim.java:413]: can we get rid of the comments?

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardDefendantClaimMatcher.java (hunk 2 above)
Comment (ElifDenizEsea) [reply src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardDefendantClaimMatcher.java:299]: there is no logic defined yet, it is auto generated

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimantClaimMatcher.java (hunk 1 above)
Comment (ElifDenizEsea) [reply src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardClaimantClaimMatcher.java]: removed

**Review by jarekPierchala — APPROVED** [review]

**Review by portizdev — DISMISSED** [review]

**Review by jarekPierchala — APPROVED** [review]

**Review by jarekPierchala — DISMISSED** [review]

**Review by sankhajuria — APPROVED** [review]

**Review by paul-pearson — APPROVED** [review]

**Review by kalachandrasekar1 — APPROVED** [review]

**Review by jarekPierchala — APPROVED** [review]

**Pull request merged**